vars/provisioningConfig.groovy

```diff
@@ -0,0 +1,12 @@
+import com.redhat.multiarch.ci.ProvisioningConfig
+
+class provisioningConfig {
+
+    def create(params) {
+        def config = new ProvisioningConfig()
+        config.KEYTABCREDENTIALID = params.KEYTABCREDENTIALID ?: 'KEYTAB'
```

Comment:
I noticed that a previous patch tried to do this in the class def - I'm guessing that didn't work?

Reply:
yeah. did not work when in class def.